packages/components/src/core/DropdownMenu/index.tsx

```diff
@@ -171,9 +173,10 @@ const DropdownMenu = <
           {...ClickAwayListenerProps}
         >
           <StyledDropdownMenuAutocompleteWrapper>
-            {title && (
+            {title && !titleElement && (
               <StyledHeaderTitle search={search}>{title}</StyledHeaderTitle>
             )}
+            {titleElement ?? null}
```

Comment:
I'm prioritizing the titleElement instead of the title, would there be a reason to prefer title or would you like this handled a different way?